### ISSUE TITLE
mTLS setting from config.yaml are not used for Dapr instances

### ISSUE THREAD

**Issue by ionut-balazs-lateral** [dapr/cli#818]
## Expected Behavior

When configuring dapr instance yaml for mTLS, I expect mTLS to be configured at runtime.
Also, it would help if the configuration would also accept a _sentryAddress_ property in the _mtls_ section.

## Actual Behavior

Updates to _config.yaml_ doesn't seem to affect the Dapr instance runtime, _mTLS_ is still disabled. The only way it works now is by supplying the flags _--enable-mtls_ and _--sentry-address_.

## Steps to Reproduce the Problem

1. Dapr 1.4.3
2. Configure mtls in config.yaml
3. Start Dapr instance
4. Monitor logs for the mTLS entry
5. sentry-address can only be set via the CLI, it would be nice to be able to do it via config

## Release Note

RELEASE NOTE: **FIX** Bug in runtime.

**Comment by Taction** [comment]
> I am using daprd directly to start the instance, not the CLI (not sure if this is relevant)

Yes, if using the Dapr CLI and with the tls enabled in `config.yaml`, tls will be enabled and `sentryAddress` will be automatically set to `localhost:50001`.

Since daprd config has tls part, daprd can use these configs if not set from flags.
This is an issue related to daprd. I'll fix that. Besides, who can help move it to the correct repo dapr/dapr?
